frontend/index.html

```diff
@@ -87,8 +87,8 @@
   <!--  Hero Banner  -->
```

Comment:
@Sharonsyra How is this page related to coursebuilder/modules/explorer/_static/components/pages/homepage.html?

Reply:
I have updated it now.